packages/utils/src/roles.js

```diff
@@ -25,13 +25,20 @@ export const isEnterpriseRole = role => ENTERPRISE_PERMISSIONS.some(permission =
  *
  * @param {object} user An authenticated user
  * @param {string} [role] Checks specified role against the role's for the user
- * @returns {boolean} true if the user has an enterprise role and/or the specified role, otherwise false
+ * @param {string} [enterprise] Checks specified role against the role's for the user for the specific enterprise
+ * @returns {boolean} true if the user has an enterprise role and/or the specified role and/or the specified role
+ * for the specific enterprise, otherwise false
  */
-export const isEnterpriseUser = (user, role) => {
+export const isEnterpriseUser = (user, role, enterprise) => {
```

Comment:
nit: Perhaps rename `enterprise` to `enterpriseUUID` so it's more explicit what the value is here.